spec/loading_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'English'
```

Comment:
Do we need this `require 'English'` ?

Reply:
I don't think it is necessary now, removed.